CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - [API Change] A new figure and axes is created (via `plt.subplots()`) when calling a plotting method with `ax=None`. Previously, the current axes was used (via `plt.gca()`) ([#211](https://github.com/ploomber/sklearn-evaluation/pull/211))
 - [Fix] Validating input elbow curve model has "score" method [#146] 
 - [Fix] Adds class labels for multi class roc plot (#209)
+- [Fix] Fixed error in which ROC curves would not graph on subplots
```

Comment:
This should be [Docs], the text should probably be, Fixed ROC curves plots to show properly.

Reply:
Yeah, maybe we can call it a fix after all. @gtduncan

Reply:
So if it fits both categories, we should probably take the code first approach.

Reply:
Something weird is happening here as well, the tests are running for 3 hours already.

Reply:
That's what I was about to ask about, but it seems to be happening to all the CI checks

Reply:
I ran all the tests locally and they passed

Reply:
Stay tuned, work on other issues :)

Reply:
the error has been fixed, please rebase